include/xgboost/c_api.h

```diff
@@ -150,20 +150,29 @@ XGB_DLL int XGBCreateEnclave(const char *enclave_image, int log_verbosity);
  */
 XGB_DLL int XGDMatrixCreateFromFile(const char *fname,
                                     int silent,
-                                    DMatrixHandle *out); 
+                                    DMatrixHandle *out);
 
 /*!
  * \brief load a data matrix from an encrypted file
  * \param fname the name of the encrypted file
  * \param silent whether print messages during loading
  * \param out a loaded data matrix
+ * \param signers list of usernames of signing clients
```

Comment:
In this file, let's add `nonce`, `nonce_size`, and `nonce_ctr` to the modified functions' param description.